bites_test.py

```diff
@@ -242,11 +242,11 @@ def test_first_player_takes_one_turn(self, render_game_mock):
     bites_game.ant_positions = starting_ant_positions
     bites_game.anthill = starting_anthill
     bites_game.anthill_food_tokens = starting_anthill_food_tokens
-    bites_game.anthill_order = [4, 3, 2, 1, 0]
+    bites_game.anthill_order = ['4', '3', '2', '1', '0']
```

Comment:
I didn't mean a list of strings. I meant a single string. 
```suggestion
    bites_game.anthill_order = 'top down'
```